tests/modules/sightings/resources/test_featured_asset_guid.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+import json
+from app.modules.sightings.models import Sighting
+from tests.modules.sightings.resources import utils as sighting_utils
+from flask import current_app
+from tests import utils
+
+
+def test_featured_asset_guid_endpoint(db, flask_app_client):
+
+    new_researcher = utils.generate_user_instance(
+        email='asset_guid_getter@mail.com', is_researcher=True
+    )
+
+    with db.session.begin():
+        db.session.add(new_researcher)
+
+    data_in = {
+        'encounters': [{}],
+        'context': 'test',
+        'locationId': 'test',
+    }
+
+    response = sighting_utils.create_sighting(
```

Comment:
This will be a lot simpler when the asset_group commit functionality is merged in. 
github.com/WildMeOrg/houston/pull/166/files#diff-41782c1591e9d509a78372e4537ad21a6d8368bfb899805004e1a2faf5834001R8-R41
but that requires PR 166 to be inspected ..... hint hint :)